steps/examples/step-gallery/README.md

```diff
@@ -0,0 +1,201 @@
+# Step Template
```

Comment:
Let's remove that content and leave it empty to denote that content needs to be added.